substrate/frame/support/procedural/src/pallet/expand/constants.rs

```diff
@@ -45,7 +47,9 @@ pub fn expand_constants(def: &mut Def) -> proc_macro2::TokenStream {
 	let config_consts = def.config.consts_metadata.iter().map(|const_| {
 		let ident = &const_.ident;
 		let const_type = &const_.type_;
-
+		let deprecation_info =
+			crate::deprecation::get_deprecation(&quote::quote! { #frame_support }, &const_.attrs)
+				.expect("invalid deprecation attribute");
```

Comment:
Is this checked in the parsing step or somewhere?
macros should not panic on wrong input